packages/a11y-base/src/tabindex-mixin.d.ts

```diff
@@ -20,7 +20,7 @@ export declare class TabindexMixinClass {
   /**
    * Indicates whether the element can be focused and where it participates in sequential keyboard navigation.
    */
-  tabindex: number | null | undefined;
+  tabIndex: number;
```

Comment:
This change is needed to avoid conflicting with standard HTML which uses `number`.